src/pages/MyPage.js

```diff
@@ -212,6 +360,20 @@ const Element = styled.div`
   font-weight: 500;
   text-align: center;
   color: ${(props) => props.theme.colors.gray};
+  @media screen and (max-width: ${(props) => props.theme.deviceSizes.mobileM}) {
+    width: 68px;
+    height: 21px;
+    margin: 16px 53px 16px 16px;
+    font-family: NotoSansCJKKR;
+    font-size: 14px;
+    font-weight: normal;
+    font-stretch: normal;
+    font-style: normal;
+    line-height: normal;
+    letter-spacing: normal;

```

Comment:
```suggestion
    font-size: 14px;
    font-weight: normal;
    line-height: normal;
```
저는 텍스트 스타일 관련해서 작성할때 line-height 속성은 간혹 기획 화면과 같이 만드는데에 있어 텍스트의 공간에 지장을 주는 속성이어서 넣을때도 있지만 아래 속성은 굳이 지정하지 않는 편입니다

- font-stretch(normal 달라지는 점이 없음) 
- font-style(이미 font-family에 처음에 지정할 때 스타일을 weight 별로 스타일을 normal로 해놨습니다), 
- letter-spacing (normal 마찬가지로 변화가 없음)

거의 모든 스타일 코드에 텍스트 속성이 있길래 공통적으로 normal로 설정이 되어 있길래 코멘트 남겨봅니다
font-family속성도 제가 reset css 집어넣을 때 body에 NotoSansCJKKR로 해놔서 따로 다시 지정해놓지 않는 편입니다

Reply:
불필요한 부분 제거완료했습니다. [d4705d0](https://github.com/BCSDLab/KOALA_FRONT_END/pull/43/commits/d4705d07593f0b30c7b2e63ce68161bdc464fb16)